ST_environs.c

```diff
@@ -231,19 +165,6 @@ static void _set_temp_reduction( void) {
     Env.temp_reduction[i] = tp[2]*tp[0] + tp[3] * (tp[0]*tp[0]);
     Env.temp_reduction[i] = max(0., Env.temp_reduction[i]);
   }
-
-#ifdef STEPWAT
-  if (UseSoilwat) {
-    if (Env.temp < 9.5 ) {
-      Env.temp_reduction[CoolSeason] = .9;
-      Env.temp_reduction[WarmSeason] = .6;
-    } else {
-      Env.temp_reduction[CoolSeason] = .6;
-      Env.temp_reduction[WarmSeason] = .9;
-    }
-  }
-#endif
-
```

Comment:
The ifdef statement should have been deleted here, but not what is within it. We need to retain:

```
if (Env.temp < 9.5 ) {
      Env.temp_reduction[CoolSeason] = .9;
      Env.temp_reduction[WarmSeason] = .6;
    } else {
      Env.temp_reduction[CoolSeason] = .6;
      Env.temp_reduction[WarmSeason] = .9;
    }
```